ShoppingCart-topic1/src/Counter.java

```diff
@@ -0,0 +1,20 @@
+public class Counter {

```

Comment:
How could you avoid problems with the counter if there are concurrent operations making transactions?